### DISCUSSION TITLE
Is there a way to start workflow request in a "suspended" mode/status?

### DISCUSSION THREAD

**Discussion by rickyschechter** [Netflix/conductor#2527] [Q&A]
I am registering the workflow with a "dynamic" definition. I will like it to be registered, stored and almost ready to be processed, but before it starts, will like to send a second request with the returned workflowId to be "/resume"d whenever I am ready.

**Comment by kishorebanala** [comment]
@rickyschechter How about using a wait task, and completing this wait task whenever the workflow is ready to progress. For example: https://netflix.github.io/conductor/labs/eventhandlers/

**Comment by rickyschechter** [comment]
@kishorebanala thank you for you reply and suggestion.
Your suggestion is interesting, although I will know when to start from "outside" workflow, for that:
1) I am wondering about a possible race condition such us, what if the WAIT task is NOT yet IN_PROGRESS (let's say a really busy Conductor Server) when the event for a complete_task is sent? would it miss the task and remain forever blocked?
2) I see a POST /event to register the handler, I do not see a POST endpoint to send the event, perhaps it is not available?

Thanks!

**Reply by aravindanr** [reply]
> I see a POST /event to register the handler, I do not see a POST endpoint to send the event, perhaps it is not available?
The event is sent using the [EVENT](https://netflix.github.io/conductor/configuration/systask/#event) task.
